src/lib.rs

```diff
@@ -1,7 +1,12 @@
 mod icons;
 pub use icons::*;
-            
-#[derive(Clone)]
+
+#[derive(Clone, Copy, PartialEq, Eq)]
```

Comment:
I added:
- `Copy`: this enum doesn't need to be heap allocated, so copy should make it less expensive than cloning.
- `PartialEq, Eq`: makes enum values comparable with equality operator.

Reply:
Nice!

Reply:
I'm making it derive [`Debug`](https://doc.rust-lang.org/std/fmt/trait.Debug.html) as well.